Decred Wallet/Features/Send/SendViewController.swift

```diff
@@ -407,8 +407,8 @@ extension SendViewController {
         let totalCost = NSDecimalNumber(value: sendAmountDcr + txFeeAndSize.fee!.dcrValue)
         let balanceAfterSending = NSDecimalNumber(value: sourceAccountBalance).subtracting(totalCost)
         
-        self.totalCostLabel.text = self.parseDCRAmount(totalCost.doubleValue)
-        self.balanceAfterSendingLabel.text = self.parseDCRAmount(balanceAfterSending.doubleValue)
+        self.totalCostLabel.text = self.parseDCRAmount(totalCost.doubleValue, usdRoundUp: 2)
+        self.balanceAfterSendingLabel.text = "\(balanceAfterSending.round(8).formattedWithSeparator) DCR"
```

Comment:
Should this not show the balance after sending in dollars?

Reply:
its not in the mock up